src/main/java/org/kohsuke/github/GHIssue.java

```diff
@@ -63,23 +63,14 @@
     protected int comments;
     @SkipFromToString
     protected String body;
-    // for backward compatibility with < 1.63, this collection needs to hold instances of Label, not GHLabel
-    protected List<Label> labels;
+    protected List<GHLabel> labels;
     protected GHUser user;
     protected String title, html_url;
     protected GHIssue.PullRequest pull_request;
     protected GHMilestone milestone;
     protected GHUser closed_by;
     protected boolean locked;
 
-    /**
-     * The type Label.
-     *
-     * @deprecated use {@link GHLabel}
-     */
-    public static class Label extends GHLabel {
```

Comment:
This is publicly visible, but not returned from any properties.  It exists in one spot (above).